kernel/cpu_vectors_aarch64.S

```diff
@@ -0,0 +1,121 @@
+/* Linkage for ARM */
+#define __ALIGN .align 2
+#define __ALIGN_STR ".align 2"
+
+#define ALIGN __ALIGN
+#define ALIGN_STR __ALIGN_STR
+
+#define ENTRY(name)                             \
+  .globl name;                                  \
+  ALIGN;                                        \
+  name:
+
+#define GLOBAL(name)                            \
+  .globl name;                                  \
+  name:
+
+#define END(name) \
+  .size name, .-name
+
+#define ENDPROC(name) \
+  .type name, %function; \
+  END(name)
+
+/*
+ * Exception trap types
+ *-----------------
+ */
+#define ET_SYNC    0
+#define ET_IRQ     1
+#define ET_FIQ     2
+#define ET_ERROR   3
+
+/*
+ * Invalid exception handlers
+ */
+    .macro	invalid_trap, el, type
+    mov	x0, #\el
+    mov x1, #\type
+    b	cpu_trap_handler
+    .endm
+
+el0_sync_invalid:
+	invalid_trap 0, ET_SYNC
+ENDPROC(el0_sync_invalid)
+
+el0_irq_invalid:
+	invalid_trap 0, ET_IRQ
+ENDPROC(el0_irq_invalid)
+
+el0_fiq_invalid:
+    invalid_trap 0, ET_FIQ
+ENDPROC(el0_fiq_invalid)
+
+el0_error_invalid:
+	invalid_trap 0, ET_ERROR
+ENDPROC(el0_error_invalid)
+
+el1_sync_invalid:
+	invalid_trap 1, ET_SYNC
+ENDPROC(el1_sync_invalid)
+
+el1_irq_invalid:
+	invalid_trap 1, ET_IRQ
+ENDPROC(el1_irq_invalid)
+
+el1_fiq_invalid:
+	invalid_trap 1, ET_FIQ
+ENDPROC(el1_fiq_invalid)
+
+el1_error_invalid:
+	invalid_trap 1, ET_ERROR
+ENDPROC(el1_error_invalid)
+
+    .macro exception_entry label
+    .align 7
+    b \label
+    .endm
+
+/*
+ * Exception vectors.
+ *
+ * AArch64 unikernel runs in EL1 mode using the SP1 stack. The vectors
+ * don't have a fixed address, only alignment (2^11) requirements.
+ */
+	.pushsection ".exception.text", "ax"
+
+    .align  11
```

Comment:
We had tried in another way to get the address of this extra link section without the change of linker script. Just like use readelf to dump the vector table symbol from elf image. But we found it's too complex. It's more complex than add a link section in linker scripts.